src/GridEngine.test.ts

```diff
@@ -1070,6 +1071,21 @@ describe("GridEngine", () => {
     expect(chars).toEqual(["player", "player2"]);
   });
 
+  it("should delegate to get all chars at position", () => {
+    mockGridTileMap.getCharactersAt = jest.fn(() => new Set([
+      { getId: () => "player" }
+    ]));
+
+    gridEngine.create(tileMapMock, {
+      characters: [{
+        id: "player",
+        sprite: playerSpriteMock,
+      }]
+    });
+    const chars = gridEngine.getCharactersAt(new Vector2(5, 4), "layer");
```

Comment:
Could you add an expect here that checks whether mockGridTileMap.getCharactersAt has been called with the correct parameters?